### ISSUE TITLE
column.isGapless should be columns.isGapless

### ISSUE THREAD

**Issue by MangelMaxime** [Dzoukr/Feliz.Bulma#8]
In Bulma `is-gapless` is applied to `columns` container and not `column`

```html
<div class="columns is-gapless">
  <div class="column">
    No gap
  </div>
  <div class="column">
    No gap
  </div>
  <div class="column">
    No gap
  </div>
  <div class="column">
    No gap
  </div>
</div>
```

**Comment by MangelMaxime** [comment]
Same thing for `column.isVcentered` and I also think it should be written `isVCentered` with a capital `C` but it's probably a personal choice :)

**Comment by Dzoukr** [comment]
Hi @MangelMaxime, I'll check it with docs. I am pretty sure there will be more of such bugs. Thanks for finding those!

**Comment by Dzoukr** [comment]
Yeah, I see more of those wrongly put in `column` instead of `columns`. Will release fix soon.

**Comment by MangelMaxime** [comment]
Awesome thank you :)

I am exploring Feliz usage and saw your Feliz.Bulma package so I am exploring it as the same time. When done, I will share my experience :)

**Comment by Dzoukr** [comment]
Finally somebody else then me using it. :))

I'll be happy to hear feadback from someone experienced like you!

**Comment by Dzoukr** [comment]
Fixed & released in 1.7.1. I hope it's ok 🤞

**Comment by MangelMaxime** [comment]
Awesome thank you :)
